googleMapsExtension/src/main/kotlin/io/github/kakaocup/kakao/gmaps/view/marker/MarkerActions.kt

```diff
@@ -18,10 +18,15 @@ interface MarkerActions : UiThread {
         }
     }
 
-    fun moveCameraTo(zoom: Float = googleMap.map.cameraPosition.zoom) = runOnUiThread{
+    fun moveCameraTo() = runOnUiThread{
         delegate.run {
-            googleMap.moveCamera(position, zoom)
+            googleMap.moveCamera(position, googleMap.map.cameraPosition.zoom)
+        }
+    }
 
+    fun moveCameraTo(zoom: Float) = runOnUiThread{
```

Comment:
Can we simplify by having:
```kotlin
fun moveCameraTo(zoom: Float? = null) = runOnUiThread {
  delegate.run {
    googleMap.moveCamera(position, zoom ?: googleMap.map.cameraPosition.zoom)
  }
}
```

Reply:
Yes, but as I remember providing nulls to function parameter is not a best practice. Also it will affect on documentation. 

Just for my point of view will be better to handle 

`
moveCameraTo()
moveCameraTo(zoom)
`

Maybe 

```
fun moveCameraTo() = runOnUiThread {
  delegate.run {
   moveCameraTo(position,googleMap.map.cameraPosition.zoom)
  }
}

fun moveCameraTo(zoom: Float) = runOnUiThread {
  delegate.run {
    googleMap.moveCamera(position, zoom)
  }
}
```
WDYT ?

Reply:
Having null by default argument is pretty much optional argument in Kotlin, no? You don't have to pass `null` yourself if you don't want to set a value. With default argument value you can decrease unnecessary verbosity. If you still think null is bad, you can use `Float.MIN_VALUE` as default value still keeping natural Kotlin overloads sugar and not having to deal with nullable.

Reply:
Passing default values - is fine to me, but except `null`s 

For example `fun moveCameraTo(zoom: Float = googleMap.map.cameraPosition.zoom)` will be fine, but unfortunately we can't call `googleMap.map.cameraPosition.zoom` from arguments because it required main thread. 

With `fun moveCameraTo(zoom: Float? = null)` it create a possibility to call it with `moveCameraTo(null)` in test code, what can create some misunderstanding 

---

My point of view is simple:
* Default values for functions 👍 
* Default `null` values for functions 👎

Reply:
Then I suggest to use default value for the function and remove duplicate.